wrappers/services.go

```diff
@@ -1585,6 +1585,7 @@ type RestartServicesFlags struct {
 // are only restarted if they are active, so if a service is meant to be
 // restarted no matter it's state, it should be included in the
 // explicitServices list.
+// The list of explicitServices needs to use systemd unit names.
```

Comment:
shouldn't we change this  to be consistent with StartServices

Reply:
We can do this, at the cost of extra remapping needed to handle unitStatues inside of RestartServices.

Reply:
Do you have strong preference to do this in this PR? I think it would be best addressed when cleaning up wrappers and introducing AppRef?

Reply:
no, this is fine for now I suppose.  Maybe a todo would have been good:

// TODO: change explicitServices format to be less unusual, more consistent  (introduce AppRef?)